frappe/charts
# fix: position of tooltip inside scroll container

Hello there!

This MR changes positining of tooltip from absolute to fixed.




###### Explanation About What Code Achieves:

Closes bug: https://github.com/frappe/charts/issues/3

###### Screenshots/GIFs:

<img width="865" alt="screen shot 2017-11-08 at 19 47 29" src="https://user-images.githubusercontent.com/5403694/32568267-0122d91e-c4be-11e7-8806-d7a86612f9b5.png">

###### Steps To Test:

Open docs/index.html, view heatmap, scroll it to the right and hover above one of the options. The tooltip should be displayed in a correct way.

###### TODOs:

  - None
